src/main/java/se/sundsvall/casestatus/integration/opene/configuration/OpenEIntegrationConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package se.sundsvall.casestatus.integration.opene.configuration;
+
+import org.springframework.cloud.openfeign.FeignBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+
+import se.sundsvall.dept44.configuration.feign.FeignConfiguration;
+import se.sundsvall.dept44.configuration.feign.FeignMultiCustomizer;
+import se.sundsvall.dept44.configuration.feign.decoder.ProblemErrorDecoder;
+
+import feign.auth.BasicAuthRequestInterceptor;
+
+@Import(FeignConfiguration.class)
+public class OpenEIntegrationConfiguration {
+
+	public static final String CLIENT_ID = "open-e";
+
+	@Bean
+	FeignBuilderCustomizer feignBuilderCustomizer(final OpenEIntegrationProperties properties) {
+		return FeignMultiCustomizer.create()
+			.withErrorDecoder(new ProblemErrorDecoder(CLIENT_ID))
```

Comment:
OpenE doesn't implement [rfc7807](https://datatracker.ietf.org/doc/html/rfc7807)

We should have a different ErrorDecoder here, right?

Reply:
IT-tests to tests this will be handled in a seperate task. The scheduled job that uses this integration is in need of some improvments.

Reply:
What task?

I can't find any tasks related to this issue?